crates/gitql-ast/src/function.rs

```diff
@@ -371,6 +380,12 @@ fn text_translate(inputs: Vec<Value>) -> Value {
 
     return Value::Text(text);
 }
+fn text_unicode(inputs: Vec<Value>) -> Value {
```

Comment:
@tbro Please fix conflicts from the previous PR and add line between the two functions